src/components/Calculator.js

```diff
@@ -0,0 +1,138 @@
+import './Calculator.css';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const ShowResult = (props) => {
```

Comment:
- [ ] (Optional) You do not need to export this component since the `Calculator` component is rendering it and they are both in the same file. You would want to do a named export when the class/function/variable is in a file and you need it in another file. 👍